socket_wrapper.hpp

```diff
@@ -1,31 +1,38 @@
-
-#include "util.h"
+#ifndef __SOCKET_WRAPPER_HPP__
```

Comment:
One more thing you need to know here (sorry, I forgot to mention this): corresponding to the C/C++ standards
```
All identifiers that begin with an underscore and either an uppercase letter or another
underscore are always reserved for any use.
```